File: iOS/Objects/src/B4iProject/LS_frmaccounts.m

#import "iCore.h"
@interface LS_frmaccounts:NSObject
@end

@implementation LS_frmaccounts

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[B4ILayoutBuilder scaleAll:views :width :height];
[B4ILayoutBuilder setScaleRate:0.0];
[views get:@"pnlloading1"].Left = (int)((50 / 100.0 * width) - ([views get:@"pnlloading1"].Width / 2));
//BA.debugLineNum = 5;BA.debugLine="pnlloading1.VerticalCenter = 50%y"[frmaccounts/General script]
[views get:@"pnlloading1"].Top = (int)((50 / 100.0 * height) - ([views get:@"pnlloading1"].Height / 2));
//BA.debugLineNum = 6;BA.debugLine="pnlaction_address.HorizontalCenter = 50%x"[frmaccounts/General script]
[views get:@"pnlaction_address"].Left = (int)((50 / 100.0 * width) - ([views get:@"pnlaction_address"].Width / 2));

}
@end

File: Android/Objects/src/com/apps/demo/designerscripts/LS_frmaccounts.java
package com.apps.demo.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmaccounts{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0d);
views.get("pnlloading1").vw.setLeft((int)((50d / 100 * width) - (views.get("pnlloading1").vw.getWidth() / 2)));
//BA.debugLineNum = 5;BA.debugLine="pnlloading1.VerticalCenter = 50%y"[frmaccounts/General script]
views.get("pnlloading1").vw.setTop((int)((50d / 100 * height) - (views.get("pnlloading1").vw.getHeight() / 2)));
//BA.debugLineNum = 6;BA.debugLine="Panel1.HorizontalCenter = 50%x"[frmaccounts/General script]
views.get("panel1").vw.setLeft((int)((50d / 100 * width) - (views.get("panel1").vw.getWidth() / 2)));

}
}